pkg/router/controller/router_controller.go

```diff
@@ -238,7 +238,6 @@ func (c *RouterController) HandleEndpointSlice(eventType watch.EventType, objMet
 			Labels:          objMeta.Labels,
 			Annotations:     objMeta.Annotations,
 			OwnerReferences: objMeta.OwnerReferences,
-			ClusterName:     objMeta.ClusterName,
```

Comment:
Just curious, where did this change come from?

Reply:
The field was deprecated in the vendor bump and not used anywhere.  I'll dig up the exact source.

Reply:
https://github.com/kubernetes/apimachinery/commit/430b920312ca0fa10eca95967764ff08f34083a3

Reply:
Ah interesting thanks!